packages/wonder-blocks-core/docs.md

```diff
@@ -1,73 +1 @@
-`wonder-blocks-core` provides building blocks for other `wonder-blocks` components.  These components, `View` and `Text` can also be used on their own.
-
-`View` roughly maps to `div` and `Text` roughly maps to `span`.
-
-These components can take styles in a variety of manners:
-
-- An inline style object
-- An `aphrodite` StyleSheet style
-- An array combining the above
-
-```js
-const {StyleSheet} = require("aphrodite");
-
-const styles = StyleSheet.create({
-    container: {
-        padding: 32,
-        backgroundColor: "plum",
-    },
-    text: {
-        fontFamily: "sans-serif",
-        fontSize: 24,
-    },
-});
-
-<View style={styles.container}>
-    <Text style={styles.text}>
-        Hello, world!
-    </Text>
-    <View style={[
-        styles.container,
-        {backgroundColor: "goldenrod", padding: 4},
-    ]}>
-        Even with an array and inline styles!
-    </View>
-</View>
-```
-
-Other props can be passed through `View` or `Text`, as if they were normal tags.
-
-```js
-<View>
-    <View onClick={() => alert("Clicked!")}>
-        Click me!
-    </View>
-
-    <Text aria-hidden="true">
-        This text is hidden from screen readers.
-    </Text>
-</View>
-```
-
-There's also an HOC that adds the same style prop to existing
-components.
-
-```js
-const {addStyle} = require("./index.js");
-const {StyleSheet} = require("aphrodite");
-
-const styles = StyleSheet.create({
-    // default style for all instances of StyledInput
-    input: {
-        fontSize: 30,
-    },
-    // style for a particular instance of StyledInput
-    pink: {
-        backgroundColor: "pink",
-    },
-});
-
-const StyledInput = addStyle("input", styles.input);
-
-<StyledInput style={styles.pink} type="text" placeholder="hello, world"/>;
-```
+`wonder-blocks-core` provides building blocks and utilities for constructing other `wonder-blocks` components. Many of these may be useful inside your applications, as well.
```

Comment:
I see, we only include introductory material to the package in docs.md.

Reply:
Yep! docs.md for examples, everything else in the file. We should probably rename docs.md to be "examples.md" or similar.